site/pages/ministerial-en.hbs

```diff
@@ -65,7 +65,7 @@
 				</div>
 				<div class="wb-feeds limit-3 col-md-4">
 					<ul class="feeds-cont list-unstyled lst-spcd">
-						<li> <a href="http://news.gc.ca/web/fd-en.do?mthd=dpt&amp;ft=atom&amp;crtr.dpt1D=6675" rel="external">Government of Canada News Releases</a> </li>
+						<li> <a href="https://news.gc.ca/web/fd-en.do?mthd=dpt&amp;ft=atom&amp;crtr.dpt1D=6675" rel="external">Government of Canada News Releases</a> </li>
```

Comment:
I think we have a new link to the news feed. All template would need to be updated.

/cc @GormFrank

Reply:
I am not sure if you are talking about a specific link, but there is indeed a "new" way to link to a department's feed through the advanced search, here is an example that brings you to all news from Foreign affairs and development: https://www.canada.ca/en/news/advanced-news-search/news-results.html?dprtmnt=departmentofforeignaffairstradeanddevelopment